packages/lpa-questionnaire-web-app/src/middleware/fetch-existing-appeal-reply.js

```diff
@@ -13,21 +10,31 @@ const {
  * @returns {Promise<*>}
  */
 module.exports = async (req, res, next) => {
-  if (!req.session) {
-    return next();
-  }
-
-  if (!req.session.appealReply || !req.session.appealReply.id) {
-    req.session.appealReply = await createOrUpdateAppealReply({ appealId: req.params.id });
+  if (!req.session || !req.params.id) {
     return next();
   }
 
+  const { id } = req.params;
   try {
-    req.log.debug({ id: req.session.appealReply.id }, 'Get existing appeal');
-    req.session.appealReply = await getExistingAppealReply(req.session.appealReply.id);
+    req.session.appealReply = await getExistingAppealReply(id);
```

Comment:
this is getting the id off the req.params
the ticket is to make it such that the id on the req.params is the one we do the lookup against
ergo this is doing the lookup by appeal

this one i perhaps -would- agree that the naming could be improved - but i think that comes from tinkering with the router to use "appealId" as the parameter name rather than "id".. 

dunno; this is 3 weeks ago, there may be reasons i didn't just go and change the router or i might have forgotten/not noticed/not bothered.. unsure.